src/js/components/Baani.js

```diff
@@ -193,8 +215,6 @@ export default class Baani extends React.PureComponent {
               onFacebookClick={this.onFacebookClick(shabad)}
               onCopyClick={this.onCopyClick(shabad)}
             />
-
-            <br />
```

Comment:
I think this was added to keep new lines in the copy pasted text.

Reply:
I checked copy paste with and without br tag, it's same as following.

```
ਤੁਧ ਬਿਨੁ ਰੋਗੁ ਰਜਾਈਆਂ ਦਾ ਓਢਣੁ ਨਾਗ ਨਿਵਾਸਾਂ ਦੇ ਰਹਣਾ ॥
tudh bin rog rajaieeaa(n) dhaa oddan naag nivaasaa(n) dhe rahanaa ||
Without Thee, the taking over of quilt is like disease and living in the house is like living with serpents
```